examples/triton/docker-compose-with-web.yml

```diff
@@ -0,0 +1,37 @@
+version: '2.1'
+
+services:
+  redis:
+    image: autopilotpattern/redis:3.2.8-r1.0.0
+    mem_limit: 4g
```

Comment:
Would this do better with either a `g4-general-4G` or even a `g4-highram-16G` package?